docs/diagrams/ListSequenceDiagram.puml

```diff
@@ -0,0 +1,76 @@
+@startuml
+!include style.puml
+
+box Ui UI_COLOR_T1
+participant ":MainWindow" as MainWindow UI_COLOR
+participant ":ResultDisplay" as ResultDisplay UI_COLOR
+end box
+[-> MainWindow : executeCommand(commandText)
+activate MainWindow
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant "c:ListOrderCommand" as ListOrderCommand LOGIC_COLOR
+participant "result:CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":ModelManager" as ModelManager MODEL_COLOR
+end box
+
+MainWindow -> LogicManager : execute(commandText)
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand(commandText)
+activate AddressBookParser
+
+create ListOrderCommand
+AddressBookParser -> ListOrderCommand
+activate ListOrderCommand
+
+ListOrderCommand --> AddressBookParser
+deactivate ListOrderCommand
+
+AddressBookParser --> LogicManager
+deactivate AddressBookParser
+
+LogicManager -> ListOrderCommand : execute()
+activate ListOrderCommand
+
+ListOrderCommand -> ModelManager : updateFilteredOrderList(PREDICATE_SHOW_ALL_ORDERS)
+activate ModelManager
+
+ModelManager -> ModelManager : setPredicate(PREDICATE_SHOW_ALL_ORDERS)
+activate ModelManager
+
+ModelManager --> ModelManager
+deactivate ModelManager
+
+ModelManager --> ListOrderCommand
+deactivate ModelManager
+|||
+create CommandResult
+ListOrderCommand -> CommandResult
+activate CommandResult
+
+CommandResult --> ListOrderCommand
+deactivate CommandResult
+destroy CommandResult
```

Comment:
You should not destroy CommandResult here as it is returned to the MainWindow. Actually, I don't think there is a need to explicitly destroy any of the objects in a sequence diagram.